.github/workflows/test-report.yaml

```
@@ -0,0 +1,47 @@
+name: Generate Longitudinal Test Report
+
+on:
+  schedule:
+    - cron: "47 6 * * *"
+  workflow_dispatch:
+
+jobs:
+  test-report:
+    name: Generate Longitudinal Test Report
```

I'm not totally sure where all this name is displayed, but if we make the previous change let's be consistent here 

```suggestion
    name: Test Report
```